### PR TITLE
Added section for listing community integrations

### PR DESCRIPTION
**Pull request by nn81**
Whisper is fantastic, many other users are already proposing integrations and extensions for different use-cases. Creating a new section helps to find these extensions and make good use of whisper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,15 @@ result = whisper.decode(model, mel, options)
 print(result.text)
 ```
 
+
+## Extensions and integrations
+
+A list of community-contributed extended functionality and integrations with whisper.
+
+[Whisper mic](https://openai.com/blog/whisper/) Allows to use use a microphone continuously
```
Comment (EliEron) [inline README.md:130]: I assume you meant to link to https://github.com/mallorbc/whisper_mic?

Comment (nn81) [reply README.md:130]: Ah, thanks! Wrong link from my copy and paste.

Comment (EliEron) [reply README.md:130]: I figured. I've made plenty of copy-paste mistakes myself :smile:.

**Comment by nn81** [comment]
Creating a new pull request with the correct data

**Pull request closed without merging**